src/app/map/map.component.svg

```diff
@@ -0,0 +1,5119 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
not sure if we'll want to just add a img tag here and import the svg from assets but this is a good stepping stone!

Reply:
I think this is **great**, within this file we can actually add styling, change the sizing and zoom. The `svg` I am working on has a lot less break points so hopefully wont be so overwhelming. 

Great work ,this is a great start.